olx.py

```diff
@@ -0,0 +1,519 @@
+            data = script.string
+            break
+    data_dict = json.loads((re.split('GPT.targeting = |;', data))[3].replace(";", ""))
+    translate = {"one": 1, "two": 2, "three": 3, "four": 4}
+    floor_number = ""
+    floor = data_dict.get("floor_select", None)
+    furniture = data_dict.get("furniture", None)
+    built_type = data_dict.get("builttype", None)
+    rooms = data_dict.get("rooms", None)
+    if floor is not None:
+        for char in data_dict.get("floor_select", None)[0]:
+            if char.isdigit():
+                floor_number += char
+        floor = int(floor_number)
+    if furniture is not None:
+        if furniture[0] == "yes":
+            furniture = True
+        else:
+            furniture = False
+    if rooms is not None:
+        rooms = translate[rooms[0]]
+    if built_type is not None:
+        built_type = built_type[0]
+    return {
+        "private_business": data_dict.get("private_business", None),
+        "floor": floor,
+        "rooms": rooms,
+        "builttype": built_type,
+        "furniture": furniture
+    }
+
+
+def parse_available_offers(markup):
+    """ Collects all offer links on search page markup
+
+    :param markup: Search page markup
+    :return: Links to offer on given search page
+    :rtype: list
+    """
+    html_parser = BeautifulSoup(markup, "html.parser")
+    not_found = html_parser.find(class_="emptynew")
+    if not_found is not None:
+        log.warning("No offers found")
+        return
+    offers = html_parser.find_all(class_='offer')
+    parsed_offers = [parse_offer_url(str(offer)) for offer in offers if offer][OFFERS_FEATURED_PER_PAGE:]
+    return parsed_offers
+
+
+def parse_offer(markup, url):
+    """ Parses data from offer page markup
+
+    :param markup: Offer page markup
+    :param url: Url of current offer page
+    :type markup: str
+    :type url: str
+    :return: Dictionary with all offer details
+    :rtype: dict
+    """
+    html_parser = BeautifulSoup(markup, "html.parser")
+    offer_content = html_parser.body
+    offer_data = parse_flat_data(str(offer_content))
+    offer_content = html_parser.find(class_='offerbody')
+    return {
+        "title": get_title(str(offer_content)),
+        "price": get_price(str(offer_content)),
+        "surface": get_surface(str(offer_content)),
+        **offer_data,
+        "description": parse_description(str(offer_content)),
+        "url": url,
+        "date": get_date_added(str(offer_content)),
+        "images": get_img_url(str(offer_content))
+    }
+
+
+def get_category(main_category, subcategory, detail_category, region, *args):
+    """ Parses all offers pages in given category
+
+    :param main_category: Main category
+    :param subcategory: Sub category
+    :param detail_category: Detail category
+    :param region: Region
+    :param args: Additional search filters
+    :return: List of all offer urls for category
+    :rtype: list
+    """
+    parsed_content = []
```

Comment:
probably you can do that in the very same way:
https://github.com/limebrains/pyotodom/pull/1/files#diff-c9b9cd160564be25826d28dccb5572a8R132